client/src/components/areaDetail.tsx

```diff
@@ -2,91 +2,126 @@ import * as React from 'react';
 import * as constants from '../data/constants';
 import * as styles from './areaDetail.module.scss';
 
+export const readablePercent = (percent: number) => {
+  return `${(percent * 100).toFixed(1)}`;
+};
+
+export const getCategorization = (percentile: number) => {
+  let categorization;
+  let categoryCircleStyle;
+
+  if (percentile >= 0.75 ) {
```

Comment:
I realized actually we now have these in variables in `constants` (`SCORE_BOUNDARY_LOW`, `SCORE_BOUNDARY_THRESHOLD`, and `SCORE_BOUNDARY_PRIORITIZED`) - can you reference those here?